src/language-raw/index.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const languages = [
+  {
+    name: "Raw (no-op)",
+    since: "1.13.0",
+    parsers: ["raw"]
+  }
+];
+
+const parsers = {
+  raw: {
+    parse(text) {
+      return { type: "Raw", text: text };
+    },
+    astFormat: "raw",
+    locStart() {
+      return 0;
+    },
+    locEnd(node) {
```

Comment:
I had to add locStart and locEnd to get the cursor-related tests passing; are these required on all parsers now? If so, we should update the documentation.

Reply:
It is documented: https://prettier.io/docs/en/plugins.html#parsers

I don't know if it was intentional. Since we need those for comments and other stuff, we just assume they are always present and call them. I guess we could do a check if they're present and disable those functionalities if they're not, but I don't know if it's worth the trouble.

Reply:
`locStart` and `locEnd` are supposed to take in a `node` and return the index in the source code where said node starts or ends, respectively.

Reply:
I understand what they're for, I just didn't think they were required.

Reply:
Looking at the docs again now, I totally misread them, my bad